### PR TITLE
Correct code blocks so they render properly in docs

### PR DESCRIPTION
**Pull request by bpaduch**
Code blocks weren't displaying correctly on ampproject.org - needed to specify language.

### CODE REVIEW

**Comment by jridgewell** [comment]
We should add a presubmit to make sure we never get a non-language-typed code block.

**Comment by dvoytenko** [comment]
@bpaduch Thanks a lot! LGTM, but one question for `text` vs `json`.

File: extensions/amp-animation/amp-animation.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ and is comprised of:
  - Timing properties
  - Keyframes
 
-```
+```text
```
Comment (bpaduch) [inline extensions/amp-animation/amp-animation.md]: I tried, it didn't render well.

Comment (bpaduch) [inline extensions/amp-animation/amp-animation.md]: I'll investigate further and put together something so we all know what formats are acceptable.  Also, Grow SDK is fixing backticks with no code specified - default to text.  Just not in the release we use.  I'll test it out when the release is available.

**Pull request merged**